### DISCUSSION TITLE
Voice assistants consistently misunderstand commands

### DISCUSSION THREAD

**Discussion by jparkhouse** [SeriousOldMan/Simulator-Controller#17] [Q&A]
Hi there,

My voice assistants are borderline unusable - they never get the command I am asking for right, and often just appear to get stuck asking some variation of 'I am here, how can I help' for several cycles, ignoring any input and then repeating, before eventually giving up.

I have a good microphone, and I have tried several different speech recognition settings in the setup wizard, but to little luck. Is there a troubleshooting guide, or some suggestions as to how to fix this?

Thanks,
Jake

**Comment by SeriousOldMan** [comment]
Hi Jake,

I am sorry, that it is not working as expected, but I am sure, we will fix that together. First, a few questions:

1. Are you using a headset or a room micorphone, for example from a webcam?
2. Do you have configured a Push-2-Talk button, for example on your steering wheel?
3. If 2 is "Yes", are you using the "Push - Talk - Release" method or the "Click - Talk - Click" method. The second method is available as a Preset and allows you to release the P2T button while talking and click it again, when you are finished.
4. Do you "understand" the Double-Press/Click for activation phrases to select between the different assistants?

Additionally:

I think you are using English recognition, right? Are you a native speaker?

There are a few users, which did not manage to get voice control working for them as good as required. All of them are using a Button Box or Stream Deck as alternative, since almost all commands can also be triggered by a push of a button. Do you have an input device like that?

Best regards

Oliver
